### DISCUSSION TITLE
DOIs missing for multiple venues

### DISCUSSION THREAD

**Discussion by stephaniebrandl** [acl-org/acl-anthology#3846] [General]
I realized that for several of my papers in the ACL anthology, there is no DOI. 
Could you please tell me where to find those?
Here is a list:
https://aclanthology.org/2022.clasp-1.2/
https://aclanthology.org/2022.aacl-short.10/
https://aclanthology.org/2024.lrec-main.580/

**Comment by mbollmann** [comment]
Hey @stephaniebrandl, I just saw this here by random chance, I’m surprised both that this “Discussions” tab exists and that I don’t get notifications for it, which I suspect is the case for everyone on the Anthology team — explaining the lack of a response.

I don’t remember if we already talked about this over Slack back then, but looking at the papers you mention:

- The AACL paper has a DOI, so I suspect it was added in the meantime.
- CLASP and LREC-COLING are not ACL events, so we don’t assign DOIs to them. It should be done by the organisers/publishers of those proceedings, if they want to have DOIs.
